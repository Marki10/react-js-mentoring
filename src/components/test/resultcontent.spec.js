import React, { Component } from 'react';
import ResultContent from '../resultcontent';
import { connect } from "react-redux";
const ReactTestRenderer = require('react-test-renderer');

describe('ResultContent', () => {
    it('Render resultcontent', () => {
        const component = ReactTestRenderer.create(
            connect(<ResultContent />)
        );
        const testTemp = component.toJSON();
        expect(testTemp).toMatchSnapshot();
    })
});