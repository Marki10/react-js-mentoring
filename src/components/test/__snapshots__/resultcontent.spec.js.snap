exports[`ResultContent Render resultcontent 1`] = `null`;
